src/types/events/base-events.ts

```diff
@@ -425,6 +426,40 @@ export interface FileUnsharedEvent {
   event_ts: string;
 }
 
+export interface FunctionParams {
+  type?: string,
+  name?: string,
+  description?: string,
+  title?: string,
+  is_required?: boolean,
+}
+
+export interface FunctionInputValues {
+  [key: string]: unknown;
+}
+
+export type FunctionOutputValues = FunctionInputValues;
+
+// TODO :: Update this with new payload info
+export interface FunctionExecutedEvent {
+  type: 'function_executed',
+  function: {
+    id: string,
+    callback_id: string,
+    title: string,
+    description: string,
+    type: string,
+    input_parameters: FunctionParams[],
+    output_parameters: FunctionParams[],
+    app_id: string,
+    date_updated: number,
+  },
+  function_execution_id: string,
+  inputs: FunctionInputValues,
+  workflow_token: string, // xwfp-xxxxxxxxxxx
```

Comment:
What do you think about naming this the same as in the payload? (`botAccessToken` I think)

Reply:
Absolutely. This is remnant I haven't cleaned up; will do so now!